test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -170,8 +170,6 @@ public override void Setup()
                 workingDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
             }
 
-            DynamoPathManager.PreloadAsmVersion("221", DynamoPathManager.Instance);
```

Comment:
@Randy-Ma that line was the only thing that was different about the Setup method in RevitSystemTestBase. I believe we can remove the Setup method and let it use the one on the base class.